docs/inputs.rst

```diff
@@ -224,15 +223,28 @@ What we call the LSST pointing database (currently simulated since Rubin Observa
 .. tip::
    The contents of the observations table in the sqlite LSST pointing database can be found `here <https://rubin-sim.lsst.io/rs_scheduler/output_schema.html>`_
 
-The latest version of rubin_sim cadence simulations can be found at https://s3df.slac.stanford.edu/data/rubin/sim-data/. An example rubin_sim simulation visualized on sky is shown below: 
+The latest version of rubin_sim cadence simulations can be found at https://s3df.slac.stanford.edu/data/rubin/sim-data/. An example rubin_sim simulation visualized on sky is shown in the plot below of the number of on-sky visits over the 10-year simulated baseline v3.2 survey: 
 
-.. raw:: html
-
-    <iframe width="700" height="360" src="https://epyc.astro.washington.edu/~lynnej/opsim_downloads/baseline_v2.0_10yrs.mp4" frameborder="0" allowfullscreen></iframe>
+.. image:: images/Rubin_v3.2_baseline_visits.png
+  :width: 410
+  :alt: Sorcha logo
+  :align: center
 
 .. attention::
    There may be changes to how this information is read in when the Rubin Observatory operations begin in ~mid-2025.
 
+Complex Physical Parameters File (Optional)
+---------------------------------------------------
+
+The complex physical parameters file is only used if you're going to include your own rotational light curve class or cometary activity class to augment the calculated apparent magnitudes. Sorcha is set up that  
+
+.. tip::
+  *  The ordering of the columns does not matter as long as the required columns exist and have entries
+  *  The first row in the complex  physical parameters file **must** list  the columns names
```

Comment:
Double single space between "complex" and "physical", also between "list" and "the". In the third bullet, the word "file" is duplicated.